Flaky test in ScimUserEndpointsMockMvcTests

### What version of UAA are you running?

develope


### How are you deploying the UAA?

I am deploying the UAA

- locally only using gradlew

./gradlew test


### What did you do?

Failing test

ScimUserEndpointsMockMvcTests > ShouldIgnoreAliasProperties > updateUser_ShouldIgnoreAliasProperties() FAILED
    java.lang.AssertionError: Status expected:<201> but was:<409>


### What did you expect to see? What goal are you trying to achieve with the UAA?

Green test if single exectued

### What did you see instead?

Failing test

Please include UAA logs if available.
See
https://github.com/cloudfoundry/uaa/actions/runs/8444191372/job/23138217071?pr=2801

--- a/uaa/src/test/java/org/cloudfoundry/identity/uaa/scim/endpoints/ScimUserEndpointsMockMvcTests.java
+++ b/uaa/src/test/java/org/cloudfoundry/identity/uaa/scim/endpoints/ScimUserEndpointsMockMvcTests.java
@@ -436,10 +436,12 @@ class ScimUserEndpointsMockMvcTests {
 
         @Test
         void updateUser_ShouldIgnoreAliasProperties() throws Exception {
+            final String email = "john.doe.%s@example.com".formatted(RandomStringUtils.randomAlphabetic(5));
+
             // create user with empty alias properties
-            final ScimUser user = new ScimUser(null, "john.doe@example.com", "Joel", "D'sa");
+            final ScimUser user = new ScimUser(null, email, "Joel", "D'sa");
             user.setPassword("password");
-            user.setPrimaryEmail("john.doe@example.com");
+            user.setPrimaryEmail(email);
             user.setAliasId(null);
             user.setAliasZid(null);
             final ScimUser createdUser = createUser(user, scimReadWriteToken, null);
@@ -469,10 +471,12 @@ class ScimUserEndpointsMockMvcTests {
 
         @Test
         void patchUser_ShouldIgnoreAliasProperties() throws Exception {
+            final String email = "john.doe.%s@example.com".formatted(RandomStringUtils.randomAlphabetic(5));
+
             // create user with empty alias properties
-            final ScimUser user = new ScimUser(null, "john.doe@example.com", "Joel", "D'sa");
+            final ScimUser user = new ScimUser(null, email, "Joel", "D'sa");
             user.setPassword("password");
-            user.setPrimaryEmail("john.doe@example.com");
+            user.setPrimaryEmail(email);
             user.setAliasId(null);
             user.setAliasZid(null);
             final ScimUser createdUser = createUser(user, scimReadWriteToken, null);

